src/vm/syncblk.cpp

```diff
@@ -2564,6 +2564,42 @@ double ComputeElapsedTimeInNanosecond(LARGE_INTEGER startTicks, LARGE_INTEGER en
     return (elapsedTicks * NsPerSecond) / freq.QuadPart;
 }
 
+DWORD AwareLock::WaitForSemEvent(INT32 timeOut)
```

Comment:
This is just extraction of part of the AwareLock::EnterEpilogHelper into a separate function to prevent mixing SEH and C++ EH in the same function.